data/json/items/comestibles/junkfood.json

```diff
@@ -1187,5 +1187,49 @@
     "charges": 4,
     "vitamins": [ [ "calcium", 4 ] ],
     "fun": 6
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "freeze_dried_meal",
+    "name": "freeze-dried meal",
+    "weight": "175 g",
+    "color": "red",
+    "spoils_in": "1461 days",
+    "container": "bag_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": -8,
+    "calories": 879,
+    "description": "A sealed plastic bag containing a full dehydrated meal.  This one contains a hearty cooked beef and egg breakfast.",
+    "price": 1200,
+    "price_postapoc": 1400,
+    "material": [ "flesh", "egg", "junk" ],
+    "volume": "250ml",
+    "charges": 1,
```

Comment:
```suggestion
```

The error noted down below actually appears to be pointed at this line (charges not being changed from default).

Reply:
Just to note, I think it points to the wrong line because the CI builds by merging the PR with the master branch, but github shows the diff to the base commit, so the line numbers might be off. I'm not sure how we can fix it, but #46353 can make it less confusing.